apps/website/src/content/docs/divider.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Divider
+description: A divider can be used to separate a layout into different sections of content or element.
+propsPath: '@itwin/itwinui-react/esm/core/Divider/Divider.d.ts'
+---
+
+<p>{frontmatter.description}</p>
+
+<LiveExample src='Divider.main.jsx' truncate={false}>
+  <AllExamples.DividerMainExample client:load />
+</LiveExample>
+
+## Usage
+
+<p>
+  The divider is set to be displayed both horizontally (by default) or vertically. Additionally, the
+  divider can be customized to suit different design needs, including adjustable styles such width
+  or height.
+</p>
+
+<LiveExample src='Divider.vertical.jsx' truncate={false}>
+  <AllExamples.DividerVerticalExample client:load />
+</LiveExample>
+
+## Props
+
+<PropsTable path={frontmatter.propsPath} />
```

Comment:
The frontmatter variable is not necessary, we can use the string directly.

```suggestion
<PropsTable path='@itwin/itwinui-react/esm/core/Divider/Divider.d.ts' />
```

We should actually remove these `propsPath` variables from all places (even in other components). Feel free to open a separate PR for that.

Reply:
I will add this as an after-PR TODO and will commit the suggested change in this PR.

Reply:
I have made the change.